posthog/temporal/batch_exports/s3_batch_export.py

```diff
@@ -166,6 +166,13 @@ def __init__(self):
         super().__init__("Endpoint URL cannot be empty.")
 
 
+class InvalidS3EndpointError(Exception):
```

Comment:
`EmptyS3EndpointURLError` is kind of redundant with this, maybe we should remove that and just raise `InvalidS3EndpointError`?